01.fizzbuzz/fizz_buzz.rb

```diff
@@ -0,0 +1,11 @@
+for num in 1..20 do
```

Comment:
動きとしては良さそうですね！

ただ Ruby では for 文を使う機会がほとんどない、かつこのやり方ならこのような書き方ができそうです！
(Slack で Range と each で書かれていましたが、それも良さそうとは思いつつ、 Ruby なら times を使うとそれっぽく感じますねー)

```suggestion
20.times.each do |num|
```


```rb
irb(main):001:0> 20.times.each { |i| p i }
0
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
=> 20
```